apps/image-editor/tests/types/type-tests.ts

```diff
@@ -258,7 +258,7 @@ imageEditor.setBrush({
   width: 20,
   color: '#FFFFFF',
 });
-imageEditor.setCropzoneRect(1 / 1);
+imageEditor.setCropzoneRatio(1 / 1);
```

Comment:
Please add `setCropzoneRect` type test.

Reply:
Could you link me to a documentation on type test? I haven't seen these before

Reply:
It means this file. `setCropzoneRect` changed to `setCropzoneRatio` so `setCropzoneRect` is missing.